src/quanteda/generate_return_series.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+
+def generate_return_series(
+    expected_annual_return, 
+    annual_volatility, 
+    length=365, 
+    freq='D', 
+    num_series=1, 
+    dist='normal', 
+    random_state=524,
+    start_date='2024-01-01'):
+    """
+    Generate a DataFrame with independent time series of returns.
+
+    Parameters
+    ----------
+    expected_annual_return : float
+        Expected annualized return as a decimal (e.g., 0.05 for 5%).
+
+    annual_volatility : float
+        Annualized volatility as a decimal (e.g., 0.2 for 20%).
+
+    length : int, default 365
```

Comment:
adopting Doris's recommendation:
```suggestion
    n_rows=365, 
```